defaults/main.yml

```diff
@@ -13,5 +13,17 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot required parameters
+letsencrypt_certbot_default_args:
+  - "certonly"
+  - "--text"
+  - "-n"
+  - "--no-self-upgrade"
+  - "--domains {{ _letsencrypt_domains | default(letsencrypt_domain) }}"
+  - "--agree-tos"
+
+
+# certbot custom arguments -- see: https://certbot.eff.org/docs/using.html
+letsencrypt_certbot_args:
+  - "--standalone"
+  - "--expand"
```

Comment:
@gronke these are the defaults as you had them -- they don't have to be touched and will work as you originally intended.